certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "MongoDB",
+    "description": "MongoDB is a general purpose, distributed, document data platform which is loved by developers and used across enterprises to run variety of workloads.You can now explore the various endpoints and perform CRUD operations and aggregations on your collection using the MongoDB Connector.",
+    "contact": {
+      "name": "MongoDB Support",
+      "url": "https://www.mongodb.com/community/forums/",
+      "email": "support-operations@mongodb.com"
+    }
+  },
+  "host": "data.mongodb-api.com",
+  "basePath": "/app/<Data API App ID>/endpoint/data/v1/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/action/insertOne": {
+      "post": {
+        "summary": "Insert Document",
+        "description": "Using the insertone endpoint, you can add a document to a collection.Use the document property in the request body to specify the document that should be created.",
```

Comment:
@DianaMDB please make sure that everything is grammatically correct and is free of typographical errors.
Do you want to keep `insertone` as a single word? Should it have casing if it's the name of the command?
Also please make sure that there are whitespaces after commas and points.

Reply:
Hi @DianaMDB!
You can keep your commands in camel case, e.g. `InsertOne` or `insertOne`.
Camel case is not suitable for string localization, but it's fine for command names if you want to keep them this way.